app/models/author.rb

```diff
@@ -4,3 +4,10 @@ class Author < ActiveRecord::Base
   validates :name, presence: true, uniqueness: true
   validates :phone_number, length: { is: 10 }
 end
+
+FactoryBot.define do
+  factory :author do
+    name "Carmen"
+    phone_number "1234567890"
+  end
+end
```

Comment:
In the future, make sure factories go inside `spec/factories`. Test logic is never the concern of the application (never inside `app/`)